react/components/CategoryFilter.js

```diff
@@ -28,13 +28,9 @@ const getSelectedCategories = rootCategory => {
   return selectedCategories
 }
 
-const CategoryFilter = ({
-  category,
-  shallow = false,
-  onCategorySelect,
-  preventRouteChange = false,
-}) => {
+const CategoryFilter = ({ category, shallow = false, onCategorySelect }) => {
   const { map } = useContext(QueryContext)
+  const preventRouteChange = false
```

Comment:
Justify that with a comment

Reply:
Why is that different from DepartmentFilter?

Reply:
why define this variable here, and not pass `false` directly?

Reply:
or define it outside of the function

Reply:
@klzns @jgfidelis  when talking to Bebber we decided that categories shouldn't support `preventRouteChange` since it doesn't really work as a facet. I can define it outside the function as a constant though.

Reply:
Why department doesnt need this code?

Reply:
What happens if you just dont pass the second param to the function?